store/postgres/src/subgraph_store.rs

```diff
@@ -228,7 +228,7 @@ impl SubgraphStore {
         }
     }
 
-    pub(crate) async fn get_proof_of_indexing(
+    pub async fn get_proof_of_indexing(
```

Comment:
It doesn't seem like there is any need for this change; `master` still compiles with this as `pub(crate)`

Reply:
Ah, I missed that; it's fine to leave it as is then